extension-admob/src/admob.cpp

```diff
@@ -206,7 +206,7 @@ static void LuaInit(lua_State* L)
     SETCONSTANT(EVENT_DESTROYED)
     SETCONSTANT(EVENT_JSON_ERROR)
     SETCONSTANT(EVENT_IMPRESSION_RECORDED)
-    SETCONSTANT(EVENT_STATUS_AUTORIZED)
```

Comment:
I mean to keep `EVENT_STATUS_AUTORIZED` here, but add it manually without SETCONSTANT macros with the same value (13)

Reply:
Yep, I did this change now.